Clinical-Genomics/cg
# No user dependency + requiered fields in Cust table

Make links between customer contacts and user table.
Make invoice_address and invoice_reference requiered in Customer model

How to test:

1 Install the no_user_dependency_on branch on rasta and on clinical-db. STAGE!!!
2 Upgrade database with scripts/make_customer_information_mandatory.sql script
3 Go to https://clinical-api-stage.scilifelab.se/admin/customer/ and try to add a new customer.
3.1 See that Invoice adress and Invoice reference now are mandatory.
3.2 See that Primary Contact, Invoice Contact and Delivery Contact now are drop downs with names defined in the user view.
